### PR TITLE
Refactor Consumers Section

### PR DESCRIPTION
**Pull request by workshur**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

- Get rid of redux
- Refactoring
- Closes #3508

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx [hunk 1]
```diff
@@ -35,38 +31,25 @@ const Details: React.FC = () => {
   const [searchParams] = useSearchParams();
   const searchValue = searchParams.get('q') || '';
   const { isReadOnly } = React.useContext(ClusterContext);
-  const { consumerGroupID, clusterName } = useAppParams<ClusterGroupParam>();
-  const dispatch = useAppDispatch();
-  const consumerGroup = useAppSelector((state) =>
-    selectById(state, consumerGroupID)
-  );
-  const isFetched = useAppSelector(getAreConsumerGroupDetailsFulfilled);
+  const routeParams = useAppParams<ClusterGroupParam>();
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/ConsumerGroups/Details/Details.tsx:34]: in my opinion de-structuring it here is better, cause maybe in the future someone might try to use `routeParams`, and get into reference and memoization problems.

**Review by Mgrdich — APPROVED** [review]

**Comment by Haarolean** [comment]
https://user-images.githubusercontent.com/1494347/226354758-bc8b7936-eda5-4115-aab8-818ec9a13b17.mov
some fonts flickering

**Review by Haarolean — CHANGES_REQUESTED** [review]
https://github.com/provectus/kafka-ui/pull/3508#issuecomment-1476238383

**Comment by workshur** [comment]
> 

I'm disabling table during fetch. The response time is high and the list might be changed

**Review by Haarolean — APPROVED** [review]

**Pull request merged**